react/w-books/src/scss/colors.scss

```diff
@@ -0,0 +1,5 @@
+$primary-color: #54AFFF;
+$secondary-color: #9FD356;
+$white: #FFF;
+$light-grey: #F5F5F5;
```

Comment:
`$light-gray`

Reply:
I've searched and it depends, in USA is Gray but in UK is Grey. I can change it anyway